PWG/EMCAL/EMCALtasks/AliEmcalCorrectionComponent.h

```diff
@@ -138,14 +136,12 @@ class AliEmcalCorrectionComponent : public TNamed {
   TString                fBasePath;                       ///< Base folder path to get root files
   TString                fCustomBadChannelFilePath;       ///< Custom path to bad channel map OADB file
 
-  Bool_t 		 fDoMergedBCs;			  ///< flag to use one histo for all BCs
-
  private:
   AliEmcalCorrectionComponent(const AliEmcalCorrectionComponent &);               // Not implemented
   AliEmcalCorrectionComponent &operator=(const AliEmcalCorrectionComponent &);    // Not implemented
   
   /// \cond CLASSIMP
-  ClassDef(AliEmcalCorrectionComponent, 7); // EMCal correction component
+  ClassDef(AliEmcalCorrectionComponent, 6); // EMCal correction component
```

Comment:
I think this needs to be bumped to 8. If someone were to run a train with today's tag, I'm fairly certain that this could cause a problem in the future (although I'm not 100% sure). However, I'm sure that it's safe to bump it to 8, so that's what I would suggest

Reply:
Hi Raymond,

Thank you for checking. Since we decided not to use setters instead use the yaml configuration, the number of parameters was reduced by one. The "fDoMergedBCs" was removed so we reduced the number of streamed parameters from 7 to 6.

Reply:
Hi Hadi,

I understand that you're removing the member - I think I need to explain in more detail. Imagine the following:

- Someone uses today's tag, which will write a `train.root` file with `AliEmcalCorrectionComponent` at 7.
- We revert to 6 in tomorrow's tag.
- Someone makes a change later, and the ClassDef is bumped to 7.
- Someone opens the `train.root` from today's tag. The streamers don't match and opening the file fails.

Admittedly, it's not super likely, but it's possible. And we can avoid the issue entirely by bumping to 8. So that's why I suggested moving to 8

Reply:
Ah OK, I see. Yes you are right. I will increment it to 8.